src/main/java/io/jenkins/plugins/opentelemetry/job/MonitoringRunListener.java

```diff
@@ -134,6 +135,12 @@ public void _onInitialize(Run run) {
                 .setAttribute(JenkinsOtelSemanticAttributes.CI_PIPELINE_RUN_NUMBER, (long) run.getNumber())
                 .setAttribute(JenkinsOtelSemanticAttributes.CI_PIPELINE_TYPE, OtelUtils.getProjectType(run));
 
+        // CULPRITS
+        if (OtelUtils.isWorkflow(run)) {
+            rootSpanBuilder
+                .setAttribute(JenkinsOtelSemanticAttributes.CI_PIPELINE_RUN_CULPRITS, ((WorkflowRun) run).getCulpritIds().stream().collect(Collectors.toList()));
```

Comment:
What does the `culpritId` hold reference for, the user email? Will it work outside Git world?

Reply:
> What does the culpritId hold reference for, the user email? Will it work outside Git world?

It's provided by Jenkins core -> https://javadoc.jenkins.io/hudson/model/User.html 

This is parsed from https://javadoc.jenkins.io/jenkins/scm/RunWithSCM.html#calculateCulprits--

In other words, any plugin that implements https://github.com/jenkinsci/scm-api-plugin/blob/d01f419bc9dcc408e834cbc333b285b1c8058e69/docs/implementation.adoc#implementing-hudsonscmscm can produce that particular output.

> Is there a way to notify the jUnit extension about this culprits? Or simply using the same code as in here would be enough?

Once we merge then we will benefit from since these attributes are for the transaction. Meanwhile, I'ld say to use the same implementation.